peerdid/peer_did.py

```diff
@@ -119,21 +125,30 @@ def create_peer_did_numalgo_2(
     return peer_did
 
 
-def resolve_peer_did(peer_did: PEER_DID, version_id=None) -> JSON:
+def resolve_peer_did(
+    peer_did: PEER_DID,
+    version_id=None,
```

Comment:
This parameter is unused

Reply:
Yes, it's added for future when we support dynamic peer dids.

Reply:
Although `versionId` is an optional part of DID DOC (https://www.w3.org/TR/did-core/#did-document-metadata), it looks like we can always specify it in the `peer_did` as DID URL parameter: https://www.w3.org/TR/did-core/#did-parameters.
So yes, I think we can remove that parameter.